src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawingAdapter.java

```diff
@@ -26,8 +27,10 @@ public void operateTo(int x, int y) {
 		ILine line = LineFactory.getBasicLine();
 		line.setStartCoordinates(this.startX, this.startY);
 		line.setEndCoordinates(x, y);
-
-		drawLine(line);
+		//drawLine(line);
+		DrawPanelController drawPanelController = DrawerFeature.getDrawerController();
```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.